project-maintainers.csv

```diff
@@ -383,9 +383,26 @@ Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos
 ,,Giedrius Statkevičius,,GiedriusS,
 ,,Povilas Versockas,,povilasv,
 ,,Dominic Green,Improbable,domgreen,
-Incubating,Flux,Hidde Beydals,Weaveworks,hiddeco,https://github.com/fluxcd/flux/blob/master/MAINTAINERS
+Incubating,Flux,Hidde Beydals,Weaveworks,hiddeco,https://github.com/fluxcd/community/blob/main/flux-project-maintainers.yaml
 ,,Michael Bridgen,Weaveworks,squaremo,
 ,,Stefan Prodan,Weaveworks,stefanprodan,
+,,Aurel Canciu, Sortlist,relu,
```

Comment:
```suggestion
,,Aurel Canciu, NexHealth,relu,
```

@relu - right?

Reply:
https://github.com/fluxcd/flux2/blob/main/MAINTAINERS might need updating as well.

Reply:
https://github.com/relu confirms.

Reply:
Yes, this is correct.

Reply:
Confirmed from Linkedin: https://www.linkedin.com/in/aurelcanciu/. Sortlist was a previous role.